src/dsp_tools/resources/start-stack/start-stack-config.yml

```diff
@@ -1,4 +1,4 @@
 ---
 
 # take commit hash of latest DSP-API release from https://github.com/dasch-swiss/dsp-api/commits/main
-DSP-API commit: cff47ed259137402c6495bb3debfb0f72b9dce5d
+DSP-API commit: ae8f44afc34081550110f6fe8867a295c2bbe3d2
```

Comment:
This is the most recent commit, but that's not what we need. We need the most recent release commit.

Reply:
Right, of course. 🤦

Reply:
```
dsp-api % git rev-parse v30.18.3
d00ac920412b8aaf653fdb996d073abd26f7f661
```